src/rad/resources/schemas/visit-1.1.0.yaml

```
@@ -0,0 +1,153 @@
+          number of points in the primary and subpixel dither
+          patterns. The array contains (X, Y) pairs of offsets in
+          the subpixel dither pattern where the X and Y
+          coordinates are defined in the ideal system. The ideal
+          coordinate system is a geomtric-distortion-corrected
+          reference frame projected onto a tangent plane. For more
+          information, see the Science Instrument Aperture File
+          (SIAF) and associated documentation. The exposure
+          identifier in meta.observation.exposure can be used to
+          index this array to obtain the relative pointing offset
+          from the initial target position for this exposure. Note
+          that the exposure identifier is 1-indexed.
+        anyOf:
+          - type: array
+            items:
+              type: number
+          - type: "null"
+        sdf:
+          special_processing: VALUE_REQUIRED
+          source:
+            origin: TBD
+        archive_catalog:
+          datatype: nvarchar(max)
+          destination: [WFIExposure.dither_executed_pattern,
+                        SourceCatalog.dither_executed_pattern,
+                        SegmentationMap.dither_executed_pattern]
+    required: [primary_name, subpixel_name, executed_pattern]
+
+
+  type:
+    title: Visit Type
+    description: |
+      The type of visit from observation planning.
+    type: string
+    enum: ['GENERAL_ENGINEERING', 'GENERIC', 'PARALLEL',
+           'PRIME_TARGETED_FIXED', 'PRIME_UNTARGETED']
+    sdf:
+      special_processing: VALUE_REQUIRED
+      source:
+        origin: PSS:dms_visit.visit_type
+    maxLength: 30
+    archive_catalog:
+      datatype: nvarchar(30)
+      destination: [WFIExposure.visit_type, GuideWindow.visit_type,
+                    SourceCatalog.visit_type, SegmentationMap.visit_type]
+  start_time:
+    title: Visit Start Time (UTC)
+    description: |
+      The UTC time at the beginning of the visit. The time
+      is serialized on disk as an International Organization for
+      Standardization (ISO) 8601-compliant ISOT string, but if opened
+      in Python with the asdf-astropy package installed, it may be
+      read as an astropy.time.Time object with all of the associated
+      methods and transforms available. See the documentation for
+      astropy.time.Time objects for more information.
+    tag: tag:stsci.edu:asdf/time/time-1.*
+    sdf:
+      special_processing: VALUE_REQUIRED
+      source:
+        origin: TBD
+    archive_catalog:
+      datatype: datetime2
+      destination: [WFIExposure.visit_start_time, GuideWindow.visit_start_time,
+                    SourceCatalog.visit_start_time, SegmentationMap.visit_start_time]
+
+  nexposures:
+    title: Number of Planned Exposures
+    description: |
+      The total number of exposures planned within the
+      visit.
+    type: integer
+    sdf:
+      special_processing: VALUE_REQUIRED
+      source:
+        origin: TBD
```

```suggestion
        origin: PSS:dms_visit.exposures_per_dither
```
Lost change from https://github.com/spacetelescope/rad/pull/571